noir-projects/aztec-nr/aztec/src/state_vars/shared_mutable/shared_mutable.nr

```diff
@@ -8,6 +8,9 @@ use crate::state_vars::{storage::Storage, shared_mutable::scheduled_value_change
 struct SharedMutable<T, DELAY> {
     context: Context,
     storage_slot: Field,
+    // The _dummy variable forces DELAY to be interpreted as a numberic value. This is a workaround to
```

Comment:
When I started reviewing I had a few seconds of bliss where I thought that this was not longer necessary and that it was fixed in noir. Rugged.